sdk/eventhub/azure-eventhub/samples/async_samples/send_async.py

```diff
@@ -16,12 +16,29 @@
 import os
 
 from azure.eventhub.aio import EventHubProducerClient
+from azure.eventhub.exceptions import EventHubError
 from azure.eventhub import EventData
 
 CONNECTION_STR = os.environ['EVENT_HUB_CONN_STR']
 EVENTHUB_NAME = os.environ['EVENT_HUB_NAME']
 
 
+async def send_event_data_list(producer):
+    # If you know beforehand that the list of events you have will not exceed the
+    # size limits, you can use the `send_batch()` api directly without creating an EventDataBatch
+
+    # Without specifying partition_id or partition_key
+    # the events will be distributed to available partitions via round-robin.
+
+    event_data_list = [EventData('Event Data {}'.format(i)) for i in range(10)]
+    try:
+        await producer.send_batch(event_data_list)
+    except ValueError:  # Size exceeds limit. This shouldn't happen if you make sure before hand.
```

Comment:
What is the message in the exception?

Reply:
And what happens if I specify both partition_id and partition_key?

Reply:
> What is the message in the exception?

Like "The list of EventData exceeds the Event Hub frame size limit. Please send a smaller list of EventData, or use EventDataBatch, which is guaranteed to be under the frame size limit"

Reply:
> And what happens if I specify both partition_id and partition_key?

The partition_id will win.

Reply:
> The partition_id will win.

Is that the behaviour currently used in the `create_batch` API?

Reply:
> > The partition_id will win.
> 
> Is that the behaviour currently used in the `create_batch` API?

Yes